src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -90,6 +90,7 @@ _ResolveAssemblies MSBuild target.
       <_AdditionalProperties>
         _ComputeFilesToPublishForRuntimeIdentifiers=true
         ;SelfContained=true
+        ;DesignTimeBuild=$(DesignTimeBuild)
```

Comment:
Does the inner-build per RID run during a design-time build? Does it even need to?

It's doing stuff related to the trimmer & AOT, it seems like should figure out how to skip it?